Enterprise-CMCS/macpro-mdct-qmr
# CMDCT-3927: removing MSC-AD measure from combined rates page

### Description
The simplest PR to remove MSC-AD from combined rates list


### Related ticket(s)
CMDCT-3927

---
### How to test
1. Log in as user with a state that has combined rates
2. Navigate to combined rates page, click on the Adult tab and make sure that MCS-AD measure is not in the list
